mne/_fiff/meas_info.py

```diff
@@ -1830,8 +1831,7 @@ def _get_chs_for_repr(self):
     @repr_html
     def _repr_html_(self, caption=None):
         """Summarize info for HTML representation."""
-        from ..html_templates import repr_templates_env
-
+        # Keep this one lazy to make _fiff fast
```

Comment:
not sure I understand this comment. we're keeping `Info._repr_html_` lazy by not nesting the import of `_get_html_template` inside it?

Reply:
crufty comment...